src/plugins/util/redis.js

```diff
@@ -12,12 +12,12 @@ const redis = {
       tags: {
         'span.kind': 'client',
         'resource.name': command,
-        'span.type': 'redis',
+        'component': 'redis',
         'db.type': 'redis',
-        'db.name': db || '0',
+        'db.instance': db || '0',
```

Comment:
given the usage in inferred services, would defaulting to `redis` be better?

Reply:
It looks like it's making the name `type:instance`, but I do see from the wiki page that type is not required. Is there any chance it will ignore `db.type` if its present?

Reply:
This is what we do for inferred services:

```java
            String remoteService = tags.get(DatabaseServiceTag.DB_INSTANCE.getTagName());
            if (Strings.isNullOrEmpty(remoteService)) {
                return null;
            }
            String dbType = tags.get(DatabaseServiceTag.DB_TYPE.getTagName());
            if (Strings.isNullOrEmpty(dbType)) {
                return remoteService;
            }
            return String.format("%s:%s", dbType, remoteService);
```

Reply:
If type is always used, then I think `redis:0` is a nicer default for the inferred service than `redis:redis`

Reply:
Can `db` actually be `nil` here? Otherwise, maybe use `unknown`?

Reply:
`db` can be `undefined`, but in the redis client that means it will default to instance `0`. If we'd rather not make that assumption based on the client behavior, I can make the change to `unknown`